content/docs/hooks-intro.md

```diff
@@ -25,86 +25,87 @@ function Example() {
+בנוסף לכתיבת קוד לשימוש חוזר וארגון קוד קשה יותר, מצאנו שמחלקות יכולות להיות מכשול גדול בלמידת React. אתה חייב להבין איך `this` עובד בJavaScript, אשר שונה מאוד מאיך שהוא עובד ברוב השפות. אתה חייב לזכור לעשות bind לevent handlers. מבלי [תחביר מוצע](https://babeljs.io/docs/en/babel-plugin-transform-class-properties/) לא יציב, הקוד מאוד מבולגן. אנשים יכולים להבין props, state ו-top-down data flow בצורה טובה אבל עדיין מתקשים עם מחלקות. ההבנה בין קומפוננטת מחלקה או פונקציה בReact ומתי להשתמש בהן הובילה לחילוקי דעות בין מתכנתי React מנוסים.
 
-Additionally, React has been out for about five years, and we want to make sure it stays relevant in the next five years. As [Svelte](https://svelte.technology/), [Angular](https://angular.io/), [Glimmer](https://glimmerjs.com/), and others show, [ahead-of-time compilation](https://en.wikipedia.org/wiki/Ahead-of-time_compilation) of components has a lot of future potential. Especially if it's not limited to templates. Recently, we've been experimenting with [component folding](https://github.com/facebook/react/issues/7323) using [Prepack](https://prepack.io/), and we've seen promising early results. However, we found that class components can encourage unintentional patterns that make these optimizations fall back to a slower path. Classes present issues for today's tools, too. For example, classes don't minify very well, and they make hot reloading flaky and unreliable. We want to present an API that makes it more likely for code to stay on the optimizable path.
+בנוסף, React כבר 5 שנים בחוץ, ואנחנו רוצים שישאר רלוונטי ל-5 שנים הבאות. כפי ש-[Svelte](https://svelte.technology/), [Angular](https://angular.io/), [Glimmer](https://glimmerjs.com/), ואחרים הציגו, [הידור מראש](https://en.wikipedia.org/wiki/Ahead-of-time_compilation) של קומפוננטות יש עתיד פוטנציאלי. במיוחד עם זה לא מגביל לתבניות. לאחרונה התחלנו להתנסות עם [component folding](https://github.com/facebook/react/issues/7323) באמצעות [Prepack](https://prepack.io/), וראינו תוצאות מוקדמות מבטיחות. למרות זאת,  מצאנו שקומפוננטת מחלקה יכולה לעודד תבניות לא מכוונות אשר יוצרות אופטימיזציה שיוצרת נתיב איטי. מחלקות מציגות את הנושאים עבור הכלים של היום. לדוגמא, מחלקות לא מצטמצמות כל כך טוב, והופכות את hot reloading ללא אמין. אנחנו רוצים להציג API אשר נותן לקוד להשאר על נתיב האופטימיזציה.
```

Comment:
I would change "[הידור מראש]" to "ל-[ahead-of-time compilation]", it is just lost in translation otherwise

Reply:
במיוחד אם זה -> במיוחד עם זה

Reply:
את הנושאים -> בעיות

Reply:
hot reloading ללא אמין -> hot reloading לשברירי ולא אמין